IMU.cpp

```diff
@@ -0,0 +1,62 @@
+#include "IMU.h"
+
+IMU::IMU()
+{
+
+  // Call imu.begin() to verify communication and initialize
+  if (imu.begin() != INV_SUCCESS)
+  {
+    while (1){
```

Comment:
You should log an error here so that you can catch an issue with the IMU in case it happens. As this is written, you can have the robot do nothing and you wouldn't know why.

Reply:
@xinkecf35 and I talked about what should happen here and there is really no "easy" way of determining if it doesn't work. We decided that we probably will have a "heartbeat" for the robot using the LED to have a visual of if something isn't working as it should as the robot won't be connected to the robot.